CHANGES.txt

```diff
@@ -175,6 +175,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Add more unit tests to internal AppendPath, PrependPath functions.
     - Add unit tests to show .filebase method strips only the last suffix if
       several apparent suffixes are present, and .suffix returns the last.
+    - Change the way the changed/unchanged target and source lists are
```

Comment:
Perhaps 

Fixes #3029 - Interactive mode incremental builds failing on windows. And then the rest of the implementation explanation? (Same for RELEASE.txt)

Reply:
reworded